src/libs/elektra/keymeta.c

```diff
@@ -158,13 +162,13 @@ int keyRewindMeta (Key * key)
 	return ksRewind (key->meta);
 }
 
-/** Iterate to the next meta information.
+/** Forwards internal iterator of metadata to the next entry
```

Comment:
Up to debate, but I think the main purpose is to get the next metadata, not to increment internal variables.
```suggestion
/** Get the next metadata entry of a Key
```